data/contributors.yaml

```
@@ -1,3 +1,13 @@
+- date: March, 2023
+  names:
+    - Federico Di Pierro
+  links:
+    - https://github.com/fededp
+  images:
+    - /img/community/contributors/federicodipierro.png
```

```suggestion
    - /img/community/contributors/1598953548691.jpeg
```
Signed-off-by: jsalinas29 <salinas.jacque@gmail.com>